javaScript/local-storage.js

```diff
@@ -0,0 +1,26 @@
+const fullName = document.getElementById('name');
+const email = document.getElementById('email');
+const message = document.getElementById('textarea');
+const button = document.getElementById('submit-button');
+
+function storeData() {
+  const nameValue = fullName.value;
+  const emailValue = email.value;
+  const messageValue = message.value;
+
+  const user = {
+    nameValue,
+    emailValue,
+    messageValue,
+  };
+
+  if (nameValue && emailValue && messageValue) {
+    const collectedData = JSON.stringify(user);
+    localStorage.setItem('user', collectedData);
+  }
```

Comment:
- [ ] Good job on preserving the data in localStorage after filling the form, and not submitting it. However, please ensure that when re-visiting the page the form inputs should be contained, be auto-populated with, the values entered in the previous visit to the page.
- [ ] Also, if at the previous visit to the page, in the same browser, no input was entered in the form then the form inputs should remain empty.